lightly/api/api_workflow_download_dataset.py

```diff
@@ -211,7 +211,7 @@ def export_label_studio_tasks_by_tag_name(
     def export_label_box_data_rows_by_tag_id(
         self,
         tag_id: str,
-    ) -> List[LabelBoxDataRow]:
+    ) -> List[Dict]:
```

Comment:
@MalteEbner made me add this all over the place.. if you really need to remove these, you'll also need to remove them from the tests and mocked stuff